src/main/kotlin/com/atlassian/performance/tools/report/api/NonparametricFacade.kt

```diff
@@ -0,0 +1,58 @@
+package com.atlassian.performance.tools.report.api
+
+import com.numericalmethod.suanshu.stats.test.rank.wilcoxon.WilcoxonRankSum
+import org.apache.commons.math3.stat.descriptive.rank.Median
+import org.apache.commons.math3.stat.inference.KolmogorovSmirnovTest
+import java.lang.Exception
+
+class NonparametricFacade(
+    private val baseline: DoubleArray,
+    private val experiment: DoubleArray,
+    private val mwAlpha: Float = 0.05F,
+    private val ksAlpha: Float = 0.05F
+) {
+
+    private val shift: Double by lazy { hodgesLehmannDistance(baseline, experiment) }
+    private val baselineMedian: Double by lazy { Median().evaluate(baseline) }
+
+    /**
+     * Make sure equalDistributionsAfterShift is TRUE before accessing this property
+     */
+    val locationShift: Double by lazy {
+        if (!equalDistributionsAfterShift) throw Exception("Distribution shapes are different at $ksAlpha significance level. Location shift is meaningless.")
```

Comment:
This line is too long. It's hard to read in the GitHub diff and in Git CLI.

Reply:
Also please do use `{` braces even when `if` body i just a single line.